site/gatsby-site/src/components/checklists/CheckListForm.js

```diff
@@ -433,6 +443,7 @@ const searchRisks = async ({
   setAllPrecedents,
```

Comment:
Please update the (very nice) function documentation above reflecting the use of the GraphQL API.